server/app.js

```diff
@@ -3,30 +3,38 @@ const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
-
+const cors = require("cors");
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 
+/*Register and SignIn*/
+const register = require("./routes/register");
+const signin = require("./routes/signin");
+
 const { json, urlencoded } = express;
 
 var app = express();
 
 app.use(logger("dev"));
 app.use(json());
+app.use(cors());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
 app.use("/", indexRouter);
 app.use("/ping", pingRouter);
+/*Register and SignIn*/
+app.use("/api", register);
```

Comment:
may be a folder structure /routes/post or something and from their have index and then we import?? also we can use new ways of importing like  ` import blah from 'blah' ` like es6. just a suggetion

Reply:
OK, I think I'll add another folder in routes to keep the register and signin routes. I see the rest of the files in server are using es5 so I'm just keep using that convention

Reply:
it upto you .... i prefer es6 but if you prefer es5 its completely fine.

Reply:
I agree on the folder structure. But let's stick to using es5 imports. On NodeJS, ES modules were experimental until version 14, so I think it's better to stick with es5 require() statements